modules/apps/portal-settings/portal-settings-web/src/main/resources/META-INF/resources/authentication/reminder_queries.jsp

```diff
@@ -0,0 +1,31 @@
+<%--
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+--%>
+
+<%@ include file="/init.jsp" %>
+
+<aui:fieldset>
+	<aui:input name="<%= Constants.CMD %>" type="hidden" value="<%= Constants.UPDATE %>" />
+
+	<aui:input helpMessage="enable-users-password-reminder-queries-help" label="enable-users-password-reminder-queries" name='<%= "settings--" + PropsKeys.USERS_REMINDER_QUERIES_ENABLED + "--" %>' type="checkbox" value="<%= PrefsPropsUtil.getBoolean(company.getCompanyId(), PropsKeys.USERS_REMINDER_QUERIES_ENABLED, PropsValues.USERS_REMINDER_QUERIES_ENABLED) %>" />
+
+	<aui:input helpMessage="enable-users-password-reminder-queries-custom-question-help" label="enable-users-password-reminder-queries-custom-question" name='<%= "settings--" + PropsKeys.USERS_REMINDER_QUERIES_CUSTOM_QUESTION_ENABLED + "--" %>' type="checkbox" value="<%= PrefsPropsUtil.getBoolean(company.getCompanyId(), PropsKeys.USERS_REMINDER_QUERIES_CUSTOM_QUESTION_ENABLED, PropsValues.USERS_REMINDER_QUERIES_CUSTOM_QUESTION_ENABLED) %>" />
+
+	<aui:input helpMessage="display-users-password-reminder-queries-answer-in-plain-text-help" label="display-users-password-reminder-queries-answer-in-plain-text" name='<%= "settings--" + PropsKeys.USERS_REMINDER_QUERIES_DISPLAY_IN_PLAIN_TEXT + "--" %>' type="checkbox" value="<%= PrefsPropsUtil.getBoolean(company.getCompanyId(), PropsKeys.USERS_REMINDER_QUERIES_DISPLAY_IN_PLAIN_TEXT, PropsValues.USERS_REMINDER_QUERIES_DISPLAY_IN_PLAIN_TEXT) %>" />
+
+	<aui:input helpMessage="require-users-password-reminder-queries-help" label="require-users-password-reminder-queries" name='<%= "settings--" + PropsKeys.USERS_REMINDER_QUERIES_REQUIRED + "--" %>' type="checkbox" value="<%= PrefsPropsUtil.getBoolean(company.getCompanyId(), PropsKeys.USERS_REMINDER_QUERIES_REQUIRED, PropsValues.USERS_REMINDER_QUERIES_REQUIRED) %>" />
+
+	<aui:input helpMessage="users-password-reminder-queries-questions-help" label="users-password-reminder-queries-questions" name='<%= "settings--" + PropsKeys.USERS_REMINDER_QUERIES_QUESTIONS + "--" %>' type="textarea" value="<%= PrefsPropsUtil.getStringArray(company.getCompanyId(), PropsKeys.USERS_REMINDER_QUERIES_QUESTIONS) %>" />

```

Comment:
@balazsmatyassyliferay when I tried deploying this module, the `compileJsp` task complained about this line. 

Also, you'll probably want to get the value as a String instead. Otherwise you'll only see the java object reference as the value.

Reply:
After fixing the compile error, I see the screen in Instance Settings.